### PR TITLE
Patch for multi disk support on Windows(TS Sourcemap)

### PR DESCRIPTION
**Pull request by iTOYS**
#492

### CODE REVIEW

File: src/debugger/sourceMapsCombinator.ts [hunk 1]
```diff
@@ -90,10 +90,21 @@ export class SourceMapsCombinator {
...
             return null;
         }
         return new SourceMapConsumer(result.map);
     }
 }
+
+// Hack for source-map-resolve and cutted disk letter
+// https://github.com/lydell/source-map-resolve/issues/9
+function readFileSync(diskLetter: string, filePath: string) {
+    return fs.readFileSync(`${diskLetter}${filePath}`);
```
Comment (vladimir-kotikov) [inline src/debugger/sourceMapsCombinator.ts]: You might want to check if the `filePath` is an ablolute path - in this case you'd broke it

Comment (iTOYS) [inline src/debugger/sourceMapsCombinator.ts]: This it's always looks like absolute and trimmed
`c:/project/tests/index.js` -> `/project/tests/index.js`
On unix like systems won't `diskLetter`

Comment (vladimir-kotikov) [inline src/debugger/sourceMapsCombinator.ts]: Yes, now it works that way, but I can imagine that when `urix` transitive dependency (which has the bug we're trying to workaround) gets updated we'd run into trouble which we could avoid by adding just one additional check

**Comment by vladimir-kotikov** [comment]
As a general request - please also add a couple of tests to cover this bug

File: src/test/debugger/sourceMapsCombinator.test.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                ],
+                "names": [],
+                "mappings": "AAAA,IAAA,MAAM,EAAM,CAAA,SAAA,CAAA,EAAA;IACR,SAAA,KAAa,CAAA,GAAA,EAAO;QACpB,IAAA,CAAA,IAAA,EAAA,GAAA;IACA;SACI,CAAA,SAAO,CAAA,SAAc,EAAA,SAAM,CAAA,EAAO;QACtC,OAAA,OAAA,EAAA,IAAA,CAAA,IAAA,EAAA,OAAA;IACJ,CAAA;;AAEA,CAAA,CAAA,CAAA,CAAA;;AAEA,OAAO,CAAC,GAAG,CAAC,KAAK,CAAC,QAAQ,CAAC,CAAC,CAAC",
+            };
+
+            let rawBundleSourcemap = { "version": 3, "sources": ["d:/hello.js"], "names": [], "mappings": "AAAA,IAAI,MAAM,EAAE,CAAC,SAAS,CAAC,EAAE;IACrB,SAAS,KAAK,CAAC,GAAG,EAAE;QAChB,IAAI,CAAC,IAAI,EAAE,GAAG;IAClB;IACA,KAAK,CAAC,SAAS,CAAC,SAAS,EAAE,SAAS,CAAC,EAAE;QACnC,OAAO,OAAO,EAAE,IAAI,CAAC,IAAI,EAAE,OAAO;IACtC,CAAC;IACD,OAAO,KAAK;AAChB,CAAC,CAAC,CAAC,CAAC;AACJ,IAAI,MAAM,EAAE,IAAI,KAAK,CAAC,gDAAgD,CAAC;AACvE,OAAO,CAAC,GAAG,CAAC,KAAK,CAAC,QAAQ,CAAC,CAAC,CAAC", "file": "hello.js", "sourceRoot": "" };
+
+            const stub = sandbox.stub(fs, "readFileSync");
+
+            stub.withArgs(pathToJS).returns(code);
```
Comment (vladimir-kotikov) [inline src/test/debugger/sourceMapsCombinator.test.ts]: Pls rename `stub` -> `fsReadFileStub`

File: src/debugger/sourceMapsCombinator.ts [hunk 3]
```diff
@@ -6,6 +6,8 @@ import * as path from "path";
 import { SourceMapConsumer, RawSourceMap, SourceMapGenerator, MappingItem, Mapping, Position, MappedPosition } from "source-map";
 import sourceMapResolve = require("source-map-resolve");
 
+const DISK: RegExp = /^[a-z]:/i;
```
Comment (vladimir-kotikov) [inline src/debugger/sourceMapsCombinator.ts]: `DISK` -> `DISK_LETTER_RE`

**Pull request merged**